trello_api.py

```diff
@@ -2,20 +2,39 @@
 import os
 from copy import deepcopy
 from to_do_item import ToDoItem
+from to_do_item import Status
+from dotenv import find_dotenv, load_dotenv
 
-API_KEY = os.getenv('API_KEY')
-TOKEN = os.getenv('TOKEN')
-BOARD_ID = os.getenv('BOARD_ID')
-TO_DO_LIST_ID = os.getenv('TO_DO_LIST_ID')
-DONE_LIST_ID = os.getenv('DONE_LIST_ID')
+TRELLO_API_BASE_URL = 'https://api.trello.com/1'
 
-CREDENTIALS = { 'key': API_KEY, 'token': TOKEN }
+TO_DO_LIST_ID = None
+DOING_LIST_ID = None
+DONE_LIST_ID = None
 
-TRELLO_API_BASE_URL = 'https://api.trello.com/1'
+def load_to_do_list_id_if_none():
+    global TO_DO_LIST_ID
+    if TO_DO_LIST_ID is None:
+        TO_DO_LIST_ID = get_list_id_by_name("To Do", os.getenv('BOARD_ID'))
```

Comment:
I did the better fix, but still had to remove the if's. This doesn't cause any performance issues, because now I only call each load list id function when I create the app.